core/src/main/kotlin/ch/ergon/dope/resolvable/expression/unaliased/type/arrayfunction/ArrayConcatExpression.kt

```diff
@@ -0,0 +1,17 @@
+package ch.ergon.dope.resolvable.expression.unaliased.type.arrayfunction
+
+import ch.ergon.dope.resolvable.expression.TypeExpression
+import ch.ergon.dope.validtype.ArrayType
+import ch.ergon.dope.validtype.ValidType
+
+class ArrayConcatExpression<T : ValidType>(
+    firstArray: TypeExpression<ArrayType<T>>,
+    secondArray: TypeExpression<ArrayType<T>>,
+    vararg arrays: TypeExpression<ArrayType<T>>,
```

Comment:
Not the biggest fan of the naming here, maybe something like `additionalArrays` or something like that.

Reply:
done